README.md

```diff
@@ -343,6 +343,90 @@ $ plextraktsync --server=Example1 info
 INFO     Enabled 2 libraries in Plex Server: ['Movies', 'TV Shows']
 ```
 
+### Per server configuration
+
+If you want to specify your config per server you can do so inside of `servers.yml`. Within the `config` part of the server configuration you can specify how that specific server should work.
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      sync:
+        plex_to_trakt:
+          collection: true
+        trakt_to_plex:
+          liked_lists: false
+```
+Both `watch` and `sync` can be used inside of the config key-value.
```

Comment:
no. only sync is supported. where you took watch is supported too?

Reply:
I think read it somewhere, but can't find that specific info I read. It could be that I misunderstood #1206 thinking it meant that watch was supported. I'll update it later today.